src/m365/teams/commands/tab/tab-get.ts

```diff
@@ -190,6 +190,7 @@ class TeamsTabGetCommand extends GraphCommand {
       };
 
       const res: TeamsTab = await request.get<TeamsTab>(requestOptions);
+      (res as any).teamsAppTabId = res.teamsApp!.id;
```

Comment:
I see we do the same thing in `teams tab list`, but according to me, this is not the right way forward. Duplicate data is not useful.
This could be useful to show in Text, CSV and MD outputs, but currently that's not covered by `cli.shouldTrimOutput()`. I'll discuss that internally. For now let's just remove the extra property from the response.